src/lib/plot/axis/axis.js

```diff
@@ -39,13 +43,17 @@ const propTypes = {
   orientation: React.PropTypes.oneOf([
     LEFT, RIGHT, TOP, BOTTOM
   ]),
+  axisType: React.PropTypes.oneOf([
+    DIRECTION.VERTICAL, DIRECTION.HORIZONTAL
+  ]).isRequired,
```

Comment:
Reading this PR in detail, could the `axisType` a derived from the `orientation`? For example, would there be a case where a `LEFT` or `RIGHT` orientation axis, would have a axisType of `HORIZONTAL`?